crates/common/src/lib.rs

```diff
@@ -30,34 +26,28 @@ pub type SdkSigner = FillProvider<
         >,
         WalletFiller<EthereumWallet>,
     >,
-    RootProvider<Http<Client>>,
-    Http<Client>,
-    Ethereum,
+    RootProvider,
 >;
 
 pub fn get_signer(key: &str, rpc_url: &str) -> SdkSigner {
     let signer = PrivateKeySigner::from_str(key).expect("wrong key ");
     let wallet = EthereumWallet::from(signer);
     let url = Url::parse(rpc_url).expect("Wrong rpc url");
-    ProviderBuilder::new()
-        .with_recommended_fillers()
-        .wallet(wallet.clone())
-        .on_http(url)
+    ProviderBuilder::new().wallet(wallet.clone()).on_http(url)
 }
 
 pub fn get_provider(rpc_url: &str) -> SdkProvider {
     let url = Url::parse(rpc_url).expect("Wrong rpc url");
-    ProviderBuilder::new()
-        .with_recommended_fillers()
-        .on_http(url)
+    ProviderBuilder::new().on_http(url)
 }
 
 #[allow(clippy::type_complexity)]
-pub async fn get_ws_provider(
-    rpc_url: &str,
-) -> Result<RootProvider<PubSubFrontend>, RpcError<TransportErrorKind>> {
+pub async fn get_ws_provider(rpc_url: &str) -> Result<RootProvider, RpcError<TransportErrorKind>> {
     let ws = WsConnect::new(rpc_url);
-    ProviderBuilder::new().on_ws(ws).await
+    ProviderBuilder::new()
+        .disable_recommended_fillers()
```

Comment:
any reason why we are disabling fillers for ws?

Reply:
There is no `with_recommended_fillers()` anymore. `ProviderBuilder::new().on_ws(ws)` is equivalent to `ProviderBuilder::new().with_recommended_fillers().on_ws(ws)` in 0.9. Anywhere I did `disable_recommended_fillers()` was just to mimic the existing behavior.

Reply:
ah , right.